### PR TITLE
Completed both files

### PR DESCRIPTION
**Pull request by janakigrad**

### CODE REVIEW

**Comment by yaminipreethik** [comment]
All changes look good. Can be merged!

**Review by yaminipreethik — APPROVED** [review]
Reviewed and looks good. Approved the PR.

**Comment by rishishah77** [comment]
Looks great. Thanks!

**Review by rishishah77 — COMMENTED** [review]
Looks good.

File: story.txt [hunk 1]
```diff
@@ -6,8 +6,9 @@ I am writing to you from a intergalactic spaceship in a alienated galaxy. I foun
 going for a ride on a Magenta rocket in October. There are 8735378765 lions and tigers of pink here! In the
 A101 there is a pool full of [11]. I fall asleep each night on a [12] bed of
 [13] and dream of [14] [15]. It feels as though I have lived here for
-[16] [17]. I hope one day you can visit, although the only way to get here now is
-[18] on a [19] [20]!!
+9 days. I hope one day you can visit, although the only way to get here now is
```
Comment (rishishah77) [inline story.txt:9]: Changes look good. Thanks!

**Pull request merged**